lib/existing-eks-nginx-observability-construct/index.ts

```diff
@@ -0,0 +1,107 @@
+// import { Construct } from 'constructs';
```

Comment:
This file is also not required, you might have to add code to existing patterns like single OSS, Existing OSS, Existing Graviton OSS to add a flag `niginx.enabled...` and add rulepaths for AmpAddon which will take care of adding rules. Also make sure you update the opentelemetry.yaml to add nginx `https://github.com/aws-observability/terraform-aws-observability-accelerator/blob/main/modules/eks-monitoring/otel-config/templates/opentelemetrycollector.yaml` config from here.

Reply:
add nginx.enabled function by referring Java PR